apps/fortuna/src/keeper.rs

```diff
@@ -2,7 +2,6 @@ use {
     crate::{
         api::{self, BlockchainState, ChainId},
         chain::{
-            eth_gas_oracle::eip1559_default_estimator,
```

Comment:
please revert all changes to this file -- it should be completely unchanged in the diff